spk/tcl/Makefile

```diff
@@ -1,29 +1,30 @@
-SPK_NAME = tcl
-SPK_VERS = 8.6.6
-SPK_REV = 1
-SPK_ICON = src/tcl.png
+SPK_NAME	= tcl
```

Comment:
Please don't indent with tabs. Use spaces if you must, but I'd rather you conform to the structure currently used in all other Makefiles. This comment applies to all the added indents in this PR.